README.md

```diff
@@ -6,3 +6,7 @@ This is the first edit after commiting the README.md
 ## Sub Heading
 
 This is the second edit that was made on the local machine to be loaded onto the hub and committed. 
+
+## Local Development
```

Comment:
added subheader....this is good for when you are working with multiple people on a repositry.